CRM/Remoteevent/RemoteEvent.php

```diff
@@ -317,4 +318,53 @@ public static function isTemplate($event_id)
             return 0;
         }
     }
+
+    /**
+     * Retrieves information about additional participants registered by the
+     * given participant. The format is the result of a Participant.autocomplete
+     * APIv4 call.
+     *
+     * @param int $participantId
+     *   The ID of the participant to retrieve additionally registered
+     *   participants for
+     *
+     * @return array
+     */
+    public static function getAdditionalParticipantInfo(int $participantId): array
+    {
+        $participant = Participant::get(FALSE)
+            ->addSelect('event_id', 'contact_id')
+            ->addWhere('id', '=', $participantId)
+            ->execute()
+            ->single();
+        $participants = CRM_Remoteevent_Registration::getRegistrations($participant['event_id'], $participant['contact_id']);
+        $additional_participant_ids = Participant::get(FALSE)
+            ->addWhere(
+                'registered_by_id',
+                'IN',
+                array_column($participants, 'id')
+            )
+            ->execute()
+            ->column('id');
+
+        if (!empty($additional_participant_ids)) {
+            $additional_participants = Participant::autocomplete(FALSE)
+                ->setIds($additional_participant_ids)
+                ->execute()
+                ->getArrayCopy();
+        }
+
+        array_walk($additional_participants, function(&$participant) {
+            $participant['message'] = implode(
+                ' ',
+                [
+                    $participant['description'][0] ?? '#' . $participant['id'],
+                    $participant['label'],
+                    '[' . ($participant['description'][1] ?? E::ts('Unknown Status')) . ']'
```

Comment:
I guess lowercase is correct, fixed. This should also use the requested locale for translation, which I now made avilable by passing the event, so that `$event->localise()` can be used instead of `E::ts()`.